Makefile

```diff
@@ -215,7 +213,7 @@ undeploy:
             fi; \
         }; \
         function delete_all_things { \
-            cmd="kubectl -n $$1 delete $$2 --all --now"; \
+            cmd="kubectl delete $$1 --all=true -A --now"; \
```

Comment:
Should we do the same for delete_thing?

delete_thing is currently used both for namespaced and non-namespaced objects, but I think `-A` should likely be fine to use even for non-namespaced.

Reply:
I think I got this one - want to check? Just as a note, I haven't tested any of the code that I wrote today. Just kinda, you know

# \#YOLO

Reply:
I'll give it a test run.